frontend/src/pages/Login.vue

```diff
@@ -5,36 +5,48 @@
         <form ref="textareaform" @submit.prevent="form" class="container">
           <div class="row">
             <div class="col mx-auto login-box">
+              <slot name="label">
+                <label class="control-label"> Select Network </label>
+              </slot>
+              <vue-select
+                class="select"
+                @input="setNetwork"
+                
```

Comment:
adding `:value="network"` here sets it to the default value instead of showing empty before any selection has been made yet

Reply:
Thanks DT. Yeah, I had that set initially but it didn't play nice when switching back and forth between mainnet. Will see if I can't fix that somehow.

Reply:
Reset backend state upon logout sorted this out. Dropdown now defaults to Mainnet.